server/opts.go

```diff
@@ -1271,9 +1271,9 @@ func parseJetStream(v interface{}, opts *Options, errors *[]error, warnings *[]e
 			switch strings.ToLower(mk) {
 			case "store_dir", "storedir":
 				opts.StoreDir = mv.(string)
-			case "max_memory_store", "max_mem_store":
+			case "max_memory_store", "max_mem_store", "max_mem":
```

Comment:
never sure why we have all these permutations, seems to just promote a situation where people cant find what they see in a config file in the docs, anyway

At account level have these:

https://github.com/nats-io/nats-server/blob/b7c79f66e49a583e1a326e290f8def8c9a4eeb8d/server/opts.go#L1205

and 

https://github.com/nats-io/nats-server/blob/b7c79f66e49a583e1a326e290f8def8c9a4eeb8d/server/opts.go#L1211

Shall I just make it the exact same everywhere? I am documenting around `max_mem` and `max_file`

Reply:
The reason is historic, every time I free form a config and it doesn't work I add that option in ;)

Reply:
We should document one way probably.